slam-libraries/viam-cartographer/src/io/read_PCD_file.h

```diff
@@ -13,14 +13,19 @@
 #include "cartographer/sensor/timed_point_cloud_data.h"
 
 namespace viam {
+static const int filenamePrefixLength = 6;
 namespace io {
 
 class ReadFile {
    public:
+    std::string time_format = "%Y-%m-%dT%H:%M:%SZ";
```

Comment:
[q] Why not have this inside the scope of `ReadTimeFromFilename` since that's where it's used?

Reply:
because other methods will probably need to know the time format, such as saving the pbstream. Should move this out of `ReadfFile` and into `io`?

Reply:
I think for now I'd add it to the function. If it needs to be pulled out in the future for other functions, it can happen then.